src/code42cli/cmds/cases.py

```diff
@@ -0,0 +1,240 @@
+import json
+import os
+from pprint import pformat
+
+import click
+from py42.clients.cases import CaseStatus
+from py42.exceptions import Py42BadRequestError
+from py42.exceptions import Py42NotFoundError
+
+from code42cli.click_ext.groups import OrderedGroup
+from code42cli.options import format_option
+from code42cli.options import sdk_options
+from code42cli.options import set_begin_default_dict
+from code42cli.options import set_end_default_dict
+from code42cli.output_formats import OutputFormatter
+
+
+case_number_arg = click.argument("case-number", type=int)
+name_option = click.option("--name", help="Name of the case.",)
+assignee_option = click.option("--assignee", help="User UID of the assignee.")
+description_option = click.option("--description", help="Description of the case.")
+notes_option = click.option("--notes", help="Notes on the case.")
+subject_option = click.option("--subject", help="User UID of a subject of the case.")
+status_option = click.option(
+    "--status",
+    help="Status of the case. `OPEN` or `CLOSED`.",
+    type=click.Choice(CaseStatus.choices()),
+)
+file_event_id_option = click.option(
+    "--event-id", required=True, help="File event id associated to the case."
+)
+
+CASES_KEYWORD = "cases"
+BEGIN_DATE_DICT = set_begin_default_dict(CASES_KEYWORD)
+END_DATE_DICT = set_end_default_dict(CASES_KEYWORD)
+
+
+def _get_cases_header():
+    return {
+        "number": "number",
```

Comment:
The rest of these headers are capitalized except for `number`

Reply:
Done.